packages/sqflite/tizen/src/constants.h

```diff
@@ -0,0 +1,68 @@
+#ifndef SQFLITE_CONSTANTS_H_
+#define SQFLITE_CONSTANTS_H_
+
+#include <string>
+
+namespace sqflite_constants {
+const std::string kPluginKey = "com.tekartik.sqflite";
```

Comment:
https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace
```suggestion
namespace sqflite_constants {

const std::string kPluginKey = "com.tekartik.sqflite";
```
The same rule applies to other files.